src/v2/view-builder/views/google-authenticator/EnrollAuthenticatorGoogleAuthenticatorView.js

```diff
@@ -0,0 +1,99 @@
+import { loc, createButton } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import EnrollGoogleAuthenticatorBarcodeView from './EnrollGoogleAuthenticatorBarcodeView';
+import EnrollAuthenticatorManualSetupView from './EnrollAuthenticatorManualSetupView';
+
+const Body = BaseForm.extend({
+  title () {
+    return loc('oie.enroll.google_authenticator.setup.title', 'login');
+  },
+
+  subtitle () {
+    return loc('oie.enroll.google_authenticator.enterCode.title', 'login');
+  },
+
+  save () {
+    return loc('mfa.challenge.verify', 'login');
+  },
+
+  className: 'oie-enroll-google-authenticator',
+
+  modelEvents: {
```

Comment:
be caution when declare modelEvents on the BaseForm child class. it will overwrite `BaseForm.modelEvents` and could cause unexpected behavior.